src/mapped-types.ts

```diff
@@ -36,6 +52,10 @@ export type NonUndefined<A> = A extends undefined ? never : A;
 /**
  * FunctionKeys
  * @desc get union type of keys that are functions in object type `T`
+ * @example
+ *   // Expect: "setName"
+ *   type MixedProps = { name: string; setName: (name: string) => void };
```

Comment:
Please add 1 empty line after MixedProps fixture for better readability. Please follow up this convention in all the examples below

Reply:
done